src/core/geometry/element/Polyline.h

```diff
@@ -18,6 +18,9 @@ template<class T>
 class Polyline : public virtual Line<T>,
                  public virtual PolylineBase {
 public:
+    static const std::size_t sizeOfCoordinates = 0;
```

Comment:
Why do we need this? Is it used? Can we remove it?

Reply:
The original code for reading elements uses sizeOfCoordinates to make the coordinates vector, which is checked for every class T in the template.  Because Polyline didn't have it, it wouldn't compile. So I had to create it. I also used it to use an alternative reading method for coordIds when the size is unknown before reading them. 

It could be changed so it always uses the new method, and the old method wouldn't be used (which is still used for elements with a fixed sizeOfCoordinates).